### PR TITLE
refactor(IVP): refactors width, scroll, toggle

### PR DESCRIPTION
**Pull request by mcoker**
* moves toggle button outside of scrolling IVP
* makes IVP narrower
* repositions icon in IVP toggle button

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1455/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by dongniwang** [comment]
Thanks @mcoker! 

I think it looks pretty good! A few things I noticed: 
- Not sure if we mentioned in the slide deck, but the "+" icon should be grey instead of black. 
- The text on the finish connection grey bar, should be black text on grey background for better contrast I think.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1459/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by dongniwang** [comment]
The other thing is - can we bring back the tooltip for the collapse/expand icon?

**Comment by mcoker** [comment]
@dongniwang made those updates.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1468/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by dongniwang** [comment]
Looks pretty good to me! Thanks! 

Assuming what we have now is PF4 tooltip (styling)?

**Comment by mcoker** [comment]
> Assuming what we have now is PF4 tooltip (styling)?

Yep!

File: app/ui-react/packages/ui/src/Integration/IntegrationVerticalFlow.tsx [hunk 1]
```diff
@@ -36,6 +37,10 @@ export class IntegrationVerticalFlow extends React.Component<
       (localStorage.getItem('iec-vertical-flow-expanded') || 'y') === 'y',
   };
 
+  public tooltipText = {
```
Comment (riccardo-forina) [inline app/ui-react/packages/ui/src/Integration/IntegrationVerticalFlow.tsx]: What’s this for?

**Comment by gashcrumb** [comment]
It doesn't seem to look as nice with a scrollbar if you're stuck with having one:

image

without that big wide expander thingie the scrollbar is less of a nuisance:

image

It does look great in the collapsed state when there's no scrollbar.

File: app/ui-react/packages/ui/src/Integration/IntegrationVerticalFlow.tsx (hunk 1 above)
Comment (mcoker) [reply app/ui-react/packages/ui/src/Integration/IntegrationVerticalFlow.tsx]: Oops, forgot to remove that.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1477/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by dongniwang** [comment]
Do we have a fixed scroll bar? Maybe we can try the hover and appear scrollbar behavior? Would that have an accessibility concern? 

It does feel like the scrollbar to the right of the IVP is less intrusive...

**Comment by gashcrumb** [comment]
That scrollbar is browser/operating system dependent, I maintain a pretty crappy linux desktop just for this kinda thing :-)

It'd probably look kinda similar on Windows 10 but I don't have that kicking around to test with.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1481/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1483/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Pull request merged**